### PR TITLE
Use official @hyperapp/html helpers

### PR DESCRIPTION
**Pull request by Swizz**
Use official @hyperapp/html helpers.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
 import "./index.css"
 
-const main = (p, c) => h("main", p, c)
-const svg = (p, c) => h("svg", p, c)
 const icon = href => svg({}, use(href))
-const h1 = (p, c) => h("h1", p, c)
-const span = (p, c) => h("span", p, c)
-const link = (p, c) => h("a", p, c)
-const button = (p, c) => h("button", p, c)
 const use = href =>
```
Comment (lukejacksonn) [inline src/index.js]: Sure wish we supported `xlink:href` so that we could do SVG (cross-browser) without concern or a custom element 😏

Comment (lukejacksonn) [reply src/index.js]: `svg([use({ href })])` _should_ work but it doesn't everywhere yet (not our fault) 😢

Comment (Swizz) [reply src/index.js]:
```js
const icon = href => svg({}, use(href))
```

Is not really a problem for me. But I think I will work on svg helpers too.
Maybe under `@hyperapp/html/svg`

Comment (lukejacksonn) [reply src/index.js]: I mean.. you should not have to use a proprietary `use` tag to have svg render consistently across browser.. it should just _work_. This is a hyperapp specific problem https://github.com/hyperapp/hyperapp/pull/211

**Comment by lukejacksonn** [comment]
I am happy to merge this.. @JorgeBucaran ?

**Pull request merged**